publisher/backfill.go

```diff
@@ -62,13 +62,13 @@ func (svc *backfillPublisher) isQueueEmpty(ctx context.Context) (bool, error) {
 	return length == 0, err
 }
 
-func (svc *backfillPublisher) updateCursor(ctx context.Context) error {
-	batch, err := svc.db.GetBlocksBatch(ctx, svc.cursor, svc.batchSize*2)
+func (svc *backfillPublisher) updateCursor(ctx context.Context, batchSize uint64) error {
+	batch, err := svc.db.GetBlocksBatch(ctx, svc.cursor, batchSize*2)
```

Comment:
Good question. I think it's so that there's an increased probability of finding `batchSize` un-indexed blocks within the range `[cursor, 2*batchSize)`.